EIPS/eip-2539.md

```diff
@@ -0,0 +1,266 @@
+---
+eip: 2539
+title: BLS12-377 curve operations
+author: Alex Vlasov (@shamatar)
+discussions-to: https://ethereum-magicians.org/t/eip-2539-bls12-377-precompile-discussion-thread/4659
+status: Draft
+type: Standards Track
+category: Core
+created: 2020-02-26
+requires: 1109, 2046
```

Comment:
```suggestion
```
Why are these required?  It feels like a chain who doesn't have these implemented would still be able to include this EIP.